frontend/packages/vkt/public/i18n/fi-FI/clerk.json

```diff
@@ -0,0 +1,26 @@
+{
+  "vkt": {
+    "component": {
+      "clerkExamEventListing": {
+        "addExaminationDate": "Lisää tutkintopäivä",
+        "header": {
+          "examDate": "Tutkintopäivä",
+          "fillings": "Paikkoja täytetty",
+          "language": "Kieli ja taso",
+          "public": "Nähtävillä julkisesti",
+          "registrationCloses": "Ilmottautuminen sulkeutuu"
+        },
+        "title": "Tutkintotilaisuudet",
```

Comment:
Figmassa `Tutkintopäivät`. `Tutkintotilaisuudet` löytyy taas Figmassa tabin otsikkona. Tämän voisi tarkistaa.

Reply:
Tästäkin laitoin kommentin vanoihin leiskoihin. Parempi tod.näk. näin mutta katsotaan mitä mieltä Otto on näistä ja muutenkin kun ennättää.